pom.xml

```diff
@@ -93,6 +94,13 @@
                 <artifactId>aws-lambda-java-events</artifactId>
                 <version>${lambda.events.version}</version>
             </dependency>
+            <dependency>
```

Comment:
By adding this, can we remove anything else?

Reply:
I'm not sure as the sdk was actually not used in other modules (xray uses different dependencies, logging does not use it at all). It's a bom, so just a way to specify dependencyManagement for all the sdk dependencies.